axelrod/tournament.py

```diff
@@ -255,8 +255,8 @@ def _run_parallel(self, processes: int=2) -> bool:
         """
         # At first sight, it might seem simpler to use the multiprocessing Pool
         # Class rather than Processes and Queues. However, this way is faster.
-        work_queue = Queue()
-        done_queue = Queue()
+        work_queue = Queue() # type: Queue
```

Comment:
nit: I think it's supposed to be two spaces before the `#`.

Reply:
Good catch. Have fixed.